### DISCUSSION TITLE
hi my second bar not set top how to i can use first task bar and scound taskbar top same time ExplorerPatcher

### DISCUSSION THREAD

**Discussion by fsashraful** [valinet/ExplorerPatcher#2598] [General]
ExplorerPahi my second bar not set top how to i can use first task bar and scound taskbar top same time ExplorerPatchertcher/ when i selected top its not working its set auto default 
![Screenshot_1](https://github.com/valinet/ExplorerPatcher/assets/39641119/174e3886-3d8f-4e8c-8b2a-e12a6bcb60fc)

**Comment by pyrates999** [comment]
Sorry, I don't understand.
